src/Core/src/Layouts/GridLayoutManager.cs

```diff
@@ -1244,6 +1256,14 @@ public Definition(GridLength gridLength)
 				}
 
 				_gridLength = gridLength;
+				_minSize = -1;
+				_maxSize = -1;
+			}
+
+			public Definition(GridLength gridLength, double minSize, double maxSize) : this(gridLength)
```

Comment:
Since this is an internal constructor, you might be able to just modify the original constructor to add the new parameters with defaults, and not have two constructors here.